bucket/mysql.json

```diff
@@ -1,5 +1,6 @@
 {
     "homepage": "https://dev.mysql.com/downloads/mysql/",
+    "description": "A freely downloadable version of the world's most popular open source database that is supported by an active community of open source developers and enthusiasts.",
```

Comment:
I do not like that `world's most popular`. Also sufficient description of this could be something like `Database`  :D 

I cannot came up with something better

Reply:
It's marketing speak, but in this case it's true. See https://db-engines.com/en/ranking_osvsc.

I shortened it a bit.